util/gconv/gconv_struct.go

```diff
@@ -388,6 +388,15 @@ func bindVarToStructAttr(structReflectValue reflect.Value, attrName string, valu
 				ReferValue: structFieldValue,
 			})
 			return
+		// Hold the time zone consistent in recursive
+		// Issue: https://github.com/gogf/gf/issues/2980
+		case "*gtime.Time", "gtime.Time":
+			doConvertWithReflectValueSet(structFieldValue, doConvertInput{
+				FromValue:  value.(interface{}),
```

Comment:
You cannot use `value.(interface{})` here as the `value` is already type of `interface{}`.